src/widgetastic_patternfly4/contextselector.py

```diff
@@ -0,0 +1,23 @@
+from . import Select
+
+
+class ContextSelector(Select):
+    ITEMS_LOCATOR = ".//ul[@class='pf-c-context-selector__menu-list']/li"
+    ITEM_LOCATOR = (".//button[contains(@class, 'pf-c-context-selector__menu-list-item')"
+                    " and normalize-space(.)={}]")
+    SEARCH_INPUT_LOCATOR = ".//input[@type='search']"
+    SEARCH_BUTTON_LOCATOR = ".//button[contains(@id, 'pf-context-selector-search-button')]"
+
+    def item_select(self, item, use_search=False):
```

Comment:
Ended up in not using search by default.
First of all - it works perfectly fine without searching, widgetastic scrolls to element and clicks it, so using search will be more time consuming with no real benefits.
Secondly, turns out clearing that searchbar isn't stable. 'Clear' button isn't there in DOM (probably is added by some css style), `self.browser.clear()` sometimes works ok, sometimes old value appear shortly after clearing, and sometimes typing text in 'cleared' field ends up in `half old string + new string + other half of old string`. 

After playing with that for 3 hours i had 70% passrate of FF and 0% on chrome and thought it's probably not worth it at this point, as it works ok without searchbar and when we need it, new version of selenium/chrome- gecko- driver might be released and the issue will be gone.